build.gradle

```diff
@@ -30,7 +30,7 @@ ext {
     joglVersion = "2.4.0-rc-20200307"
     javalinVersion = "5.6.2"
     photonGlDriverLibVersion = "dev-v2023.1.0-9-g75fc678"
-    rknnVersion = "dev-v2024.0.0-30-g001b5ec"
+    rknnVersion = "dev-v2024.0.0-62-g71b3f3f"
```

Comment:
Does this parameter actually effect anything? may be unused or the other one needs to be removed and just refernece this

Reply:
Yeah we should centralize it here.
![image](https://github.com/PhotonVision/photonvision/assets/29715865/98e9a6e4-ddb2-4c7e-be79-3d8ecb3ee02c)